assets/model_monitoring/components/tests/unit/conftest.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Unit test level fixtures."""
+
+import os
+import pytest
+import zipfile
+import time
+import random
+import string
+from spark_mltable import SPARK_ZIP_PATH
+
+
+@pytest.fixture(scope="session")
+def code_zip_test_setup():
```

Comment:
this is to fix the module not found error in executors when run UT remotely?

Reply:
Yes. It should already be added by Ilya's PR so idk why it says I'm modifying it in the diff. I'll fix that.